tests.py

```diff
@@ -153,36 +189,37 @@ def fn():
 
     def test_create_authentication_request(self):
         api = toopher.ToopherApi('key', 'secret')
+        pairing_id = str(uuid.uuid4())
```

Comment:
It's because of how we differentiate between usernames and IDs in the authenticate method. We try and convert the param to a UUID to see if it's a pairing_id, and if that causes an exception, we treat it as a username.